packages/ethereum-storage/src/gas-price-definer.ts

```diff
@@ -65,31 +64,52 @@ export default class GasPriceDefiner {
         ]);
       }
     }
-
+    else if (
```

Comment:
The format makes the function very difficult to read. @GrandGarcon please ensure you have your prettier well configured.

Reply:
my apologies ,  i have now configured  my  vscode and config file to automatically insure consistency .